components/sections/Navbar/index.tsx

```diff
@@ -0,0 +1,76 @@
+import * as AccessibleIcon from '@radix-ui/react-accessible-icon';
+import Image from 'next/image';
+import { useTheme } from 'next-themes';
+import { useMemo } from 'react';
+
+import LanguageDropdown from '@/components/Common/LanguageDropDown';
+import ThemeToggle from '@/components/Common/ThemeToggle';
+import NavItem from '@/components/sections/NavItem';
+import { useLocale } from '@/hooks/useLocale';
+import { useNavigation } from '@/hooks/useNavigation';
+
+import style from './index.module.css';
+
+const Navbar: React.FC = () => {
+  const { newNavigationItems } = useNavigation();
+  const { availableLocales, currentLocale } = useLocale();
+  const { theme, setTheme } = useTheme();
+
+  const GHLogoSrc = `/static/images/logos/social-github${
```

Comment:
Also you shouldn't use this to decide which image to render.

I recommend doing what we did here: https://github.com/nodejs/nodejs.org/blob/main/components/Common/ThemeToggle/index.tsx#L25-L26

Rendering the two images, but using Tailwind Classes to hide one or another depending of the current Theme :)

Reply:
Thank you for this suggestion. This really simplifies a lot of things.

Reply:
Right? :)